library/core/src/intrinsics.rs

```diff
@@ -1914,10 +1914,17 @@ extern "rust-intrinsic" {
     #[rustc_const_unstable(feature = "const_raw_ptr_comparison", issue = "53020")]
     pub fn ptr_guaranteed_ne<T>(ptr: *const T, other: *const T) -> bool;
 
-    /// Allocate at compile time. Should not be called at runtime.
+    /// Allocate at compile time.
+    /// Returns a null pointer at runtime.
```

Comment:
This doesn't match the impl. I guess returning null is also ok, let's do that

Reply:
I fixed the impl one.